lib/Agrammon/Documentation.pm6

```diff
@@ -18,7 +19,7 @@ sub get-model-data( Agrammon::Model $model, $sort, :%technical! ) {
 }
 
 #| Create a LaTeX source of the model
-sub create-latex-source( Str $model-name, Agrammon::Model $model, $sort, :%technical! ) is export {
+sub create-latex-source( Str $model-name, Agrammon::Model $model, $sort, $doc-sections, :%technical! ) is export {
```

Comment:
The call to this in `documentation.t` needs updating

Reply:
Oh, silly, ...